### PR TITLE
fix broken unit test

### PR DESCRIPTION
**Pull request by Bachstelze**
The current build failed due to a 404 url.

### CODE REVIEW

**Comment by codelucas** [comment]
Great fix, I was wondering why the recent builds were having issues

File: tests/unit_tests.py [hunk 1]
```diff
@@ -753,8 +753,7 @@ class TestDownloadPdf(unittest.TestCase):
     @print_test
     def test_article_pdf_ignoring(self):
         empty_pdf = "%PDF-"  # empty PDF constant
-        a = Article(url='http://www.technik-medien.at/ePaper_Download/'
-                        'IoT4Industry+Business_2018-10-31_2018-03.pdf',
+        a = Article(url='https://www.adobe.com/pdf/pdfs/ISO32000-1PublicPatentLicense.pdf',
```
Comment (codelucas) [inline tests/unit_tests.py:756]: how did you find this url, ideally we should move all these test resources offline

**Comment by codelucas** [comment]
done

File: tests/unit_tests.py (hunk 1 above)
Comment (Bachstelze) [reply tests/unit_tests.py:756]: It is pdf url that is already used in the unit tests. Yes, it is a bit random if the tests depend on the availability of online resources. But to a good degree the project is about this attribute.

**Pull request merged**